mne/decoding/transformer.py

```diff
@@ -113,12 +113,15 @@ def __init__(self, info=None, scalings=None, with_mean=True,
 
         if not (scalings is None or isinstance(scalings, (dict, str))):
             raise ValueError('scalings type should be dict, str, or None, '
-                             'got %s' % type(scalings))
+                             'got {}'.format(type(scalings)))
         if isinstance(scalings, string_types) and \
                 scalings not in ('mean', 'median'):
             raise ValueError('Invalid method for scaling, must be "mean" or '
-                             '"median" but got %s' % scalings)
+                             '"median" but got {}'.format(scalings))
         if scalings is None or isinstance(scalings, dict):
+            if info is None:
+                raise ValueError('Need to specify "info" if scalings is ' +
+                                 str(type(scalings)))
```

Comment:
By convention we usually use `%` unless there is a good reason to use `.format` and rarely use `+`, but this should be fine

Reply:
agreed. Please avoid + and use string formatting.